tools/recetox_msfinder/macros.xml

```diff
@@ -0,0 +1,60 @@
+<macros>
+    <token name="@TOOL_VERSION@">0.1.0</token>
+    <xml name="requirements">
+        <requirements>
+            <container type="docker">zargham12/msfinder:november</container>
+        </requirements>
+    </xml>
+    <xml name="creator">
+        <creator>
+            <person
+                givenName="Zargham"
+                familyName="Ahmad"
+                url="https://github.com/zargham-ahmad"
+                identifier="0000-0002-6096-224X" />
+            <organization
+                url="https://www.recetox.muni.cz/"
+                email="GalaxyToolsDevelopmentandDeployment@space.muni.cz"
+                name="RECETOX MUNI" />
+        </creator>
+    </xml>
+    <xml name="input">
+        <param name="input_data" type="data" format="msp,mat">
+            <label>Input file to process</label>
+        </param>
+    </xml>
+    <xml name="parameter">
+        <param type="float" value="0.01" name="Ms2Tolerance" label="Ms2Tolerance" help="The mass tolerance for matching experimental- and reference fragments." />
```

Comment:
I'd suggest something like "Fragments m/z tolerance" for a label.

Reply:
@hechth can you share your thoughts on this

Reply:
I think this should be mass tolerance used for fragment peak matching and annotation.